### PR TITLE
Create "Messaging Tweaks" extension

### PR DESCRIPTION
**Pull request by Wolvan**
A few useful tweaks for the new Tumblr IM.

Fullfills one point of #748

### CODE REVIEW

**Comment by hobinjk** [comment]
I can't get my messages to display on the left by disabling the option to move them to the right

**Comment by hobinjk** [comment]
The minimization by clicking the title doesn't appear to suppress the click event, causing the side blog thing to open (preventDefault and stopPropagation should do the trick)

**Comment by Wolvan** [comment]
@hobinjk I just realized the thing with the click event myself, thanks for the hint though. The thing with disabling I must have accidentally broken mid-developement. Thanks a lot for telling me

File: Extensions/messaging_tweaks.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.tab.iconic.tab_messaging .true-icon {
+	background: url("http://i.imgur.com/OAoKqzx.png") no-repeat !important;
```
Comment (ThePsionic) [inline Extensions/messaging_tweaks.css]: I'd rather this URL is base64'd.

File: Extensions/messaging_tweaks.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+			text: "Tumbley the Tumblr",
+			default: false,
+			value: false
+		},
+		"hide_chat_bubble": {
+			text: "Hide the chat bubble completely",
+			default: false,
+			value: false
+		},
+
```
Comment (ThePsionic) [inline Extensions/messaging_tweaks.js]: I love nitpicking about seemingly useless empty lines.

**Comment by ThePsionic** [comment]
I appear to be unable to open the control panel for this extension.
a
The problem persists throughout refreshes.

File: Extensions/messaging_tweaks.js (hunk 2 above)
Comment (Wolvan) [inline Extensions/messaging_tweaks.js]: The lines are there to group what is under which seperator

**Comment by ThePsionic** [comment]
Update: The problem is fixed by removing the empty lines in the preferences as well as removing the trailing comma in the preference list.

I'm not quite sure what you expected to happen when setting the default for the colors to `rgba(0,0,0,.1)`:
a
Why don't you just parse a hex color?

File: Extensions/messaging_tweaks.js (hunk 2 above)
Comment (ThePsionic) [inline Extensions/messaging_tweaks.js]: They might just be bad, see my comment.

File: Extensions/messaging_tweaks.js [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+			type: "text",
+			default: "rgba(0,0,0,.1)",
+			value: "rgba(0,0,0,.1)"
+		},
+		"background_override": {
+			text: "Change chat background",
+			type: "text",
+			default: "",
+			value: ""
+		},
```
Comment (ThePsionic) [inline Extensions/messaging_tweaks.js]: Remove this trailing comma.

**Comment by ThePsionic** [comment]
An idea: Remove the ability to click on the blog names in the title if the option to hide on title click is on.

**Comment by Wolvan** [comment]
@ThePsionic said it looks good and I can r+ for him in gitter
@homu r=ThePsionic

**Pull request merged**